conda/cli/main.py

```diff
@@ -63,6 +65,12 @@ def main_subshell(*args, post_parse_hook=None, **kwargs):
         "verbosity": pre_args.verbosity,
     }
 
+    try:
+        # load extra parameters defined by plugins
+        context.plugin_manager.load_configuration_parameters()
+    except PluginError as exc:
+        logging.warning(exc.message)
```

Comment:
Not really sure what to do here about the logging call. Below is a call to `init_loggers` but this method really needs to be called as soon as possible in this function.

Ideally, I would move the `init_loggers` call to the top and then create a `logger` object inside this function to properly log. Maybe that's not necessary though 🤷. I would appreciate some feedback here on this.

Reply:
Plugins implementing this functionality will require similar access to a fully initialized logging system, so we don't accidentally allow them to register the default output. This is a sign of a tightly coupled system sadly, so the solution might be to make context initialization lazier and move to a later stage. I'm afraid this effort may require some refactoring in the config system essentially.